services/metadata/backend/data/db.json

```diff
@@ -505,7 +505,7 @@
                         {
                             "place": {
                                 "name": "Geonames",
-                                "url": "https://www.geonames.org/2661604"
+                                "url": "https://www.geonames.org/2661604/basel.html"
```

Comment:
did you add those manually? I'm not sure we can guarantee too have the proper name of the place all the time

Reply:
Correct, in the case of Geonames it's most probably possible to have it like that, but the other services don't contain anything good to display. This is definitely something we need to talk about.